### PR TITLE
[CAPZ] update dashboard to show the coverage results

### PR DESCRIPTION
**Pull request by cpanato**
per discussion in the [slack thread](https://kubernetes.slack.com/archives/C09QZ4DQB/p1597965863004900?thread_ts=1597936542.000400&cid=C09QZ4DQB), we need to update the config to show the coverage properly in the dashboards.

This PR update the config based on the discussion.

Can you help to review @MushuEE ? thanks!
/cc @CecileRobertMichon @nader-ziada

### CODE REVIEW

**Comment by cpanato** [comment]
also did some changes to follow this PR: https://github.com/kubernetes/test-infra/pull/18945

File: config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml [hunk 1]
```diff
@@ -328,14 +328,11 @@ presubmits:
           cp coverage.* ${ARTIFACTS}
           cd ../../k8s.io/test-infra/gopherage
           GO111MODULE=on go build .
-          find ${ARTIFACTS} -type f -iname 'junit*.xml' -exec rm {} \;
-          ./gopherage filter --exclude-path="zz_generated,generated\.pb\.go" "${ARTIFACTS}/coverage.out" > "${ARTIFACTS}/filtered.cov" || result=$?
+          ./gopherage filter --exclude-path="zz_generated,generated\.go" "${ARTIFACTS}/coverage.out" > "${ARTIFACTS}/filtered.cov" || result=$?
           ./gopherage html "${ARTIFACTS}/filtered.cov" > "${ARTIFACTS}/coverage.html" || result=$?
           ./gopherage junit --threshold 0.05 "${ARTIFACTS}/filtered.cov" > "${ARTIFACTS}/junit_coverage.xml" || result=$?
```
Comment (CecileRobertMichon) [inline config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml:333]: should the threshold also be 0 to match the periodic job?

Comment (cpanato) [reply config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml:333]: sure!

Comment (cpanato) [reply config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml]: done

**Comment by cpanato** [comment]
@fabriziopandini can you take a look when you have some time? thanks!

also @CecileRobertMichon @nader-ziada ptal

**Review by CecileRobertMichon — APPROVED** [review]
/lgtm
/approve

**Comment by cpanato** [comment]
rebased and fix the conflict, need again a lgtm @CecileRobertMichon

**Pull request merged**